uportal-war/src/main/java/org/jasig/portal/rest/PortletsRESTController.java

```diff
@@ -55,24 +62,22 @@
 @Controller
 public class PortletsRESTController {
 
+    @Autowired
     private IPortletDefinitionRegistry portletDefinitionRegistry;
+
+    @Autowired
     private IPortletCategoryRegistry portletCategoryRegistry;
-    private IPersonManager personManager;
 
     @Autowired
-    public void setPortletDefinitionRegistry(IPortletDefinitionRegistry portletDefinitionRegistry) {
```

Comment:
Worth keeping the public setter method in the spirit of maintaining the exposed public API of this public class?